test/db/cmd/cmd_i

```diff
@@ -4138,7 +4138,7 @@ vaddr      paddr      target     type   name
 0x00001ff0 4 reloc.__gmon_start
 0x00001ff8 4 reloc._Jv_RegisterClasses
 0x00001ffc 4 reloc._ITM_registerTMCloneTable
-0x00002000 16 section..got.plt
+0x00002000 0 section..got.plt
```

Comment:
why this change? is this a bug

Reply:
When there is no section information, sections are created with DT_DYNAMIC information. However there is no infomation about the .plt.got length. So i choose 0.

Reply:
makes sense